src/metabase/api/card.clj

```diff
@@ -629,8 +629,10 @@
         card  (api/read-check (Card card-id))
         query (-> (assoc (query-for-card card parameters constraints middleware)
                          :async? true)
-                  (update :middleware merge {:js-int-to-string?      true
-                                             :ignore-cached-results? ignore_cache}))
+                  (update :middleware (fn [middleware]
+                                        (merge
```

Comment:
why the swap of the merge order here?

Reply:
The `middleware` argument should take precedence since we need `js-int-to-string?` to be false for exports, so that pks/fks aren't converted to strings.

In the current code, the passed-in `middleware` is always overwritten by the hard-coded values here.